src/celeritas/mat/MaterialParams.cc

```diff
@@ -24,6 +24,7 @@
 #include "celeritas/io/ImportData.hh"
 
 #include "MaterialData.hh"  // IWYU pragma: associated
+
```

Comment:
Looks like one of these days might be worth running a clang-format everywhere...

Reply:
@sethrj, @stognini Thank you for quick reviews and approvals!